src/pages/Contacts.jsx

```diff
@@ -0,0 +1,83 @@
+// React Imports
+import React, { useState } from 'react';
+// Material UI Imports
+import AddIcon from '@mui/icons-material/Add';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+// Component Imports
+import { useContactsList } from '@hooks';
+import { AddContactModal, DeleteContactModal } from '@components/Modals';
+import { ContactListTable } from '@components/Contacts';
+import { LoadingAnimation, EmptyListNotification } from '@components/Notification';
+
+/**
+ * Contacts Component - Component that generates Contacts Page for PASS
+ *
+ * @memberof Pages
+ * @name Contacts
+ */
+
```

Comment:
Could remove the spacing between the JSDoc and `Contacts` page. Would need to include a `@returns` for JSDoc.

Reply:
fixed